data_prep/introspector.py

```diff
@@ -40,6 +40,7 @@
 INTROSPECTOR_ENDPOINT = ''
 INTROSPECTOR_CFG = ''
 INTROSPECTOR_FUNCTION = ''
+INTROSPECTOR_ORACLE_1 = ''
```

Comment:
Nitpicking: Could you please make this name more self-explanatory? E.g., `INTROSPECTOR_ORACLE_KEYWORD`, like the API name.
Thanks!

Reply:
Done, used your suggestion and also change the `INTROSPECTOR_FUNCTION` to `INTROSPECTOR_ORACLE_FAR_REACH`